frontend/language/src/nb.json

```diff
@@ -1720,17 +1720,20 @@
   "ux_editor.no_components_selected": "Velg en side for å se forhåndsvisningen",
   "ux_editor.no_text": "Ingen tekst",
   "ux_editor.open_preview": "Åpne forhåndsvisning",
-  "ux_editor.options.code_list_only": "Denne komponenten støtter bare oppsett med forhåndsdefinerte kodelister.",
   "ux_editor.options.code_list_referenceId.description": "Her kan du legge til en referanse-ID til en dynamisk kodeliste som er satt opp i koden.",
   "ux_editor.options.code_list_referenceId.description_details": "Du bruker dynamiske kodelister for å tilpasse alternativer for brukerne. Det kan for eksempel være tilpasninger ut fra geografisk plassering, eller valg brukeren gjør tidligere i skjemaet.",
+  "ux_editor.options.modal_header_library_code_list": "Kodeliste fra biblioteket",
+  "ux_editor.options.modal_header_manual_code_list": "Kodeliste",
   "ux_editor.options.multiple": "{{value}} alternativer",
   "ux_editor.options.option_edit_text": "Rediger kodeliste",
-  "ux_editor.options.option_remove_text": "Fjern valgt kodeliste",
+  "ux_editor.options.option_remove_text": "Slett valgt kodeliste",
   "ux_editor.options.section_heading": "Valg for kodelister",
   "ux_editor.options.single": "{{value}} alternativ",
   "ux_editor.options.tab_code_list": "Velg kodeliste",
   "ux_editor.options.tab_manual": "Sett opp egne alternativer",
+  "ux_editor.options.tab_option_list_alert_title": "Kan ikke åpne Velg kodeliste fordi det er oppgitt en ID på fanen Angi referanse-ID. Slett denne referansen og gå tilbake for å velge kodeliste fra biblioteket.",
```

Comment:
Would it not be easier for the user if he/she did not have to go back to remove the text on the referenceID to be able to add an option list? I think this alert should be placed in the component that lists all the actions for adding an option list and the text should be changed to something like: `Komponenten refererer allerede til en egendefinert ID. Legger du til en ny kodeliste her vil du overstyre den egendefinerte ID-en.`

Reply:
This will take some work. I've made a commit with the other changes so far and will work on this and the comment above